packages/standalone-analyzer/src/index.ts

```diff
@@ -0,0 +1,92 @@
+import {
+  normalizeOptions,
+  Output,
+  type Options,
+  type NormalizedOptions,
+} from "@codecov/bundler-plugin-core";
+import { PLUGIN_NAME, PLUGIN_VERSION } from "./version";
+import { normalizeStandaloneOptions, type StandaloneOptions } from "./options";
+import { getAssets } from "./assets";
+
+/**
+ * Generates a Codecov bundle stats report and optionally uploads it to Codecov. This function can
```

Comment:
This is in a format that can go to `typedoc` like screenshot below
![Screenshot 2024-09-03 at 5 40 07 PM](https://github.com/user-attachments/assets/ce22313d-1ae2-41bb-b674-105370702fc4)